database.go

```diff
@@ -1139,9 +1137,9 @@ func (db *database) dropMeasurement(name string) error {
 }
 
 // dropSeries will delete all data with the seriesID
-func (rp *RetentionPolicy) dropSeries(seriesID uint32) error {
+func (rp *RetentionPolicy) dropSeries(seriesIDs ...uint32) error {
```

Comment:
So, idiomatic is becoming a 4 letter word in the go community, but yes, it is to support sending a single series ID in the future.

This way, your signature for drop series would be `dropSeries(id)` as opposed to `dropSeries([]SeriesIDs{id})`.

This is very common to see in the standard library as well.